crate/cli/src/actions/google/identities/delete_identities.rs

```diff
@@ -0,0 +1,24 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+
+use super::IDENTITIES_ENDPOINT;
+use crate::{actions::google::gmail_client::GmailClient, error::CliError};
+
+/// Deletes a client-side encryption identity. The authenticated user can no longer use the identity to send encrypted messages. You cannot restore the identity after you delete it. Instead, use the identities.create method to create another identity with the same configuration.
+#[derive(Parser)]
+#[clap(verbatim_doc_comment)]
+pub struct DeleteIdentitiesAction {
+    /// The primary email address associated with the client-side encryption identity configuration that's retrieved.
+    #[clap(long = "user-id", short = 'u', required = true)]
+    user_id: String,
```

Comment:
Is it an ID or an email? Is an e-mail an ID?

Reply:
this id is an email from KMS point of view - but in Google's API documentation it is called a user ID